test/test_pages/js_instrument/instrument_pyside.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Test page for JS Instrumentation configured python side.</title>
+  </head>
+  <body>
+    <h3>Test page for JS Instrumentation configured python side.</h3>
+    <p>
+      Because the instrumentation occurs before the page loads, we can not instrument
+      objects that we subsequently setup in this html. So we interact with standard
+      window objects and record the response.
+    </p>
+    <script type="text/javascript">
+        // Doubles as a test for the webdriver property
+        console.log(window.navigator.webdriver);
+        // Test set a cookie
+        window.document.cookie = 'a=COOKIE';
+        // Test get a cookie
+        console.log(window.document.cookie);
+        // Test both fetch() and window.fetch()
+        fetch(
+          "https://example1.com"
```

Comment:
These are real websites. We should avoid hitting them in our test suite.

I recommend using `example.com` and `example.org` (since these domains are official test domains by iana), but should still avoid making requests to the live servers. We can do that by setting the pref `network.dns.localDomains = example.com,example.org` which forces these to resolve to localhost.

Reply:
Fixed in e2ef603.